### DISCUSSION TITLE
Meta: Remove some discussion categories

### DISCUSSION THREAD

**Discussion by davetapley** [mobxjs/mobx-state-tree#1961] [Ideas] [answered]
I created https://github.com/mobxjs/mobx-state-tree/discussions/1959 as a 'discussion' with 'Ideas' category but subsequently discovered when you go to open an issue there are template for **Feature request** and **Conceptual question** categories, and so did a "create issue from discussion" (https://github.com/mobxjs/mobx-state-tree/issues/1960), but now we have an discussion **and** and issue for the same idea/feature request 🙈 

Can we remove the 'Ideas' discussion category?

**Comment by coolsoftwaretyler** [comment, marked as answer]
I would also love to see a de-duplication of this category, but I'd advocate to keep the ideas as discussions, so that the issue tracker is more specifically reserved for bugs and concrete feature requests.

**Reply by coolsoftwaretyler** [reply]
Hoping to make this change with https://github.com/mobxjs/mobx-state-tree/pull/2017
